src/content/docs/network-performance-monitoring/troubleshooting/snmp-walk.mdx

```diff
@@ -11,63 +11,75 @@ redirects:
+    title="SNMP v3 example"
+  >
+  ```bash
+  snmpwalk -v 3 -On -l $LEVEL -u $USERNAME -a $AUTH_PROTOCOL -A $AUTH_PASSPHRASE -x $PRIV_PROTOCOL -X $PRIV_PASSPHRASE $IP_ADDRESS .1.3.6.1.2.1.1.2.0
+  ```
 
-```bash
-snmpwalk -v 3 -On -l $LEVEL -u $USERNAME -a $AUTH_PROTOCOL -A $AUTH_PASSPHRASE -x $PRIV_PROTOCOL -X $PRIV_PASSPHRASE $IP_ADDRESS .1.3.6.1.2.1.1.2.0
-```
-Replacing the following settings as necessary: 
+  Replacing the following settings as necessary: 
 
-```
-$LEVEL == noAuthNoPriv | authNoPriv | authPriv
-$USERNAME == SNMP v3 User
-$AUTH_PROTOCOL == MD5 | SHA
-$AUTH_PASSPHRASE == Authentication passphrase
-$PRIV_PROTOCOL == DES | AES
-$PRIV_PASSPHRASE == Privacy passphrase
-$IP_ADDRESS == target device IP
-```
+  ```bash
+  $LEVEL == noAuthNoPriv | authNoPriv | authPriv
+  $USERNAME == SNMP v3 User
+  $AUTH_PROTOCOL == MD5 | SHA
+  $AUTH_PASSPHRASE == Authentication passphrase
+  $PRIV_PROTOCOL == DES | AES
+  $PRIV_PASSPHRASE == Privacy passphrase
+  $IP_ADDRESS == target device IP
+  ```
 
-The expected output from this command is something similar to this in your console: 
+  The following is an example of the expected output for this command: 
 
-```
-.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.9.1.46
-```
+  ```bash
+  .1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.9.1.46
+  ```
+  </Collapser>
+</CollapserGroup>
 
-### Capturing Full SNMP Walk
+### Capturing full SNMP walk [#capturing-snmpwalk]
 
-It is often desirable to capture the output of walking every OID available on your device(s). This output is especially used when creating new SNMP profiles for `ktranslate` and is a required artifact when [opening a profile request](https://github.com/kentik/snmp-profiles/issues/new/choose) on the GitHub repo.
+You may want to capture the output of walking every OID available on your devices. This output is used when creating new SNMP profiles for `ktranslate`, and it's a requirement to [open a profile request on GitHub](https://github.com/kentik/snmp-profiles/issues/new/choose).
 
-The primary differences in your command for this are changing the target to a root `.` and redirecting the output to a file that you can gather data from later. 
+The primary differences in your command for this are changing the target to a root `.`, and redirecting the output to a file that you can gather data from later. 
 
-**SNMP v2 Example:**
+**SNMP v2 and v3 example**
 
 ```bash
 snmpwalk -v 2c -On -c $COMMUNITY $IP_ADDRESS . >> snmpwalk.out
 ```
 
-*SNMP v3 has the same required changes*
+The output of this command will be a file named `snmpwalk.out`, that lists every OID that the device responds to.
 
-The output of this command will be a file named `snmpwalk.out` that lists every OID that the device responds to. Note on devices with a large number of interfaces it is expected for this walk to take upwards of 10+ minutes to complete. 
+<Callout variant="tip">
+Devices with a large number of interfaces, this walk takes more than 10 minutes to complete. 
```

Comment:
replace: 

`Devices with...` >> `On devices with...`

`this walk takes...` >> `SNMP walks can take...`